app-emulation/aranym/aranym-1.0.2.recipe

```diff
@@ -0,0 +1,59 @@
+SUMMARY="Atari Running on Any Machine"
+DESCRIPTION="ARAnyM (Atari Running on Any Machine) is a multiplatform virtual \
+machine for running Atari ST/TT/Falcon operating systems and applications."
+HOMEPAGE="http://github.com/aranym/aranym"
+COPYRIGHT="2001-2018 ARAnyM developer team"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/aranym/aranym/archive/ARANYM_${portVersion//./_}.tar.gz"
+CHECKSUM_SHA256="c0935aee14cca55c08e75a94b77dcd69f326ae563ae371b22c7b9d8efd82e24e"
+SOURCE_DIR="aranym-ARANYM_${portVersion//./_}"
+PATCHES="aranym-$portVersion.patch"
```

Comment:
It would be nice to use the patchset format, as haikuporter can generate it for you.
```diff
-PATCHES="aranym-$portVersion.patch"
+PATCHES="aranym-$portVersion.patchset"
```
Suggestion on how to proceed:
```
haikuporter -c aranym
haikuporter -B aranym
sed -i -e "s/^PATCHES=/#&/" aranym-1.0.2.recipe
cd patches
mv aranym-1.0.2.patch aranym-1.0.2.patch.old
haikuporter -e aranym
head -4 aranym-1.0.2.patch.old | sed -e "4 s/[PATCH] //" aranym-1.0.2.patch.old > aranym-1.0.2.patchset.new
tail -n +5 aranym-1.0.2.patchset >>aranym-1.0.2.patchset.new
diff aranym-1.0.2.patch.old aranym-1.0.2.patchset.new
mv aranym-1.0.2.patchset aranym-1.0.2.patchset.old
cp aranym-1.0.2.patchset.new aranym-1.0.2.patch
git mv aranym-1.0.2.patch aranym-1.0.2.patchset
cd ..
sed -i -e "s/^#\(PATCHES=\)/\1/" -e "/PATCHES/ s/patch\>/&set/" aranym-1.0.2.recipe
git commit -m "aranym: switch to patchset format" aranym-1.0.2.recipe \
	patches/aranym-1.0.2.patch patches/aranym-1.0.2.patchset
```
Sorry for giving detailed instructions. I'm sure you would have found what to do if I had explained this in english.
I did so because it's much easier (for me) to write Unix commands than explain a non trivial process in english :)

BTW, it builds fine on x86_64. I haven't tried yet on the other architetures.
(I'll do so later.)